src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LOGO = " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n" + "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String HELLO_MESSAGE = " Hello! I'm Duke\n" + "     What can I do for you?\n";
+    private static final String BYE_MESSAGE = " Bye. Hope to see you again soon!";
+    private static final String SEPARATOR = "____________________________________________________________";
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String COMMAND_LIST_WORD = "list";
+    private static final String COMMAND_MARK_WORD = "mark";
+    private static final String COMMAND_UNMARK_WORD = "unmark";
+    private static final String COMMAND_EXIT_WORD = "bye";
+    private static final String LINE_PREFIX = "    ";
+
+    private static final Task[] taskList = new Task[100];
```

Comment:
You can change the variable name to all in capital letters because the type is final